tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/team/mailboxes/TeamMailboxCallbackUtils.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.james.jmap.team.mailboxes
+
+import com.linagora.tmail.james.jmap.contact.{ContactFields, EmailAddressContactSearchEngine}
+import com.linagora.tmail.team.TeamMailbox
+import javax.inject.Inject
+import org.reactivestreams.Publisher
+import reactor.core.scala.publisher.SMono
+
+class TeamMailboxCallbackUtils @Inject()(contactSearchEngine: EmailAddressContactSearchEngine) {
```

Comment:
Nicer.

But the name sucks.

And that is not yet a generic API with a multibinder.